tests/test_Matcher.py

```diff
@@ -0,0 +1,80 @@
+from os import remove
+from prioritizator import Matcher
+from prioritizator import Task
+from shutil import copyfile
+
+WHITE_WINS = 1
+DRAW = 0.5
+WHITE_LOSES = 0
+
+
+def test_constructor_0_arguments():
+    matcher = Matcher()
+    obtained_data_path = matcher.data_path
+    expected_data_path = None
+    assert obtained_data_path == expected_data_path
+
+
+def test_setting_data_path():
+    data_path = "tests/test_data/test_load_task_list.csv"
+    matcher = Matcher()
+    matcher.data_path = data_path
+    obtained_data_path = matcher.data_path
+    expected_data_path = data_path
+    assert obtained_data_path == expected_data_path
```

Comment:
Estás tres líneas son código repetido con las 30-32. Por favor, refactoriza para DRY. Tal vez también puedas agregar la 28 (aunque la 28 la necesitamos en la 29) y la 19.

La función quedaría algo así:
```suggestion
def assert_data_path(matcher, data_path):
    obtained_data_path = matcher.data_path
    expected_data_path = data_path
    assert obtained_data_path == expected_data_path
```